Agreements/Airport/Ant_AA.txt

```diff
@@ -1,2 +1,2 @@
 COPX:*:*:*:EGAA:*:Antrim:AAAPP:*:10000:|BEL
-COPX:*:*:*:EGAC:*:Antrim:AAAPP:*:10000:|MAGEE
```

Comment:
MAGEE reflects the end point of all routes to EGAC in the SRD - I would prefer we kept this as is.

Reply:
Yeah all routes terminate at MAGEE so prefer to keep it in. Frustrating having the route draw go all the way to the airport when they'll never get that far before being vectored.

Reply:
> It works perfectly as intended here, The names, im not sure what you area people use to remember stuff so im happy to take any advice

BEL is beyond EGAC :) An aircraft will have missed Belfast City if they get to BEL! MAGEE is fine - it's where all routes into EGAC terminate at if you look at the SRD. Please leave as is :)